splinterd/src/main.rs

```diff
@@ -164,6 +164,14 @@ fn main() {
             .takes_value(true),
     );
 
+    #[cfg(feature = "database")]
+    let app = app.arg(
+        Arg::with_name("database")
+            .long("database")
+            .long_help("DB connection url, postgres://db_admin:db_password@127.0.0.1:5432/db")
```

Comment:
Suggest removing the example URL here; need to keep these short to cut down on wrapping.

Reply:
updated to remove example url